x-pack/legacy/plugins/index_management/public/app/services/notification.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { NotificationsStart } from '../../../../../../../src/core/public';
+
+class NotificationService {
+  private toasts: any;
+
+  public init(notifications: NotificationsStart): void {
+    this.toasts = notifications.toasts;
+  }
+
+  private addToasts = (title: string, type: 'danger' | 'warning' | 'success', text?: string) => {
+    this.toasts.add({
+      title,
+      color: type,
+      text,
+    });
+  };
+
+  public showDangerToast(title: string, text?: string) {
+    this.addToasts(title, 'danger', text);
+  }
+
+  public showWarningToast(title: string, text?: string) {
+    this.addToasts(title, 'warning', text);
+  }
+
+  public showSuccessToast(title: string, text?: string) {
+    this.addToasts(title, 'success', text);
```

Comment:
nit: looks like the toasts service API has methods like `addSuccess` we could use (I think it may add an icon too - so if it's not what we want ignore this entirely).

Same for above `warning` and `danger` instances.

Reply:
Ah, I somehow missed that. I'll look into it - thanks!

Reply:
I added this as an item to fix for the next phase of NP work.